arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,6 +410,11 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current state of Analysis Services resource. It allows more custom terminal state like Paused."
```

Comment:
@shihying - In that case, lets make sure the description of these two properties in clear in how it can be used. Also, lets have these as enums with the possible values listed for both "state" and "provisioningState". Such things help generate better docs from the swagger and easier to use SDKs.